integration-tests/rpc_integration_test.go

```diff
@@ -127,12 +127,9 @@ func runGetOrdersTest(t *testing.T, rpcEndpointPrefix, rpcServerType string, rpc
 	require.NoError(t, err)
 
 	// Create 10 new valid orders.
-	// TODO(albrow): Update this when scenario supports creating orders in a batch.
 	numOrders := 10
-	signedTestOrders := make([]*zeroex.SignedOrder, numOrders)
-	for i := 0; i < numOrders; i++ {
-		signedTestOrders[i] = scenario.NewSignedTestOrder(t, orderopts.SetupMakerState(true))
-	}
+	orderOptions := scenario.OptionsForAll(orderopts.SetupMakerState(true))
+	signedTestOrders := scenario.NewSignedTestOrdersBatch(t, numOrders, orderOptions)
```

Comment:
Would it make sense to encapsulate the `time.Sleep` call in `NewSignedTestOrder` and `NewSignedTestOrdersBatch`?

Reply:
It's not quite that simple. There are some places where we don't need to call `time.Sleep` and have a more efficient solution outside of the `scenario` package (e.g. most of the tests in __order_watcher_test.go__). There is probably room for improvement here and we might be able to take some parts of the solution from __order_watcher_test.go__ and use them everywhere with a few tweaks, but that is probably best suited for a separate PR.

Reply:
This is already being tracked in https://github.com/0xProject/0x-mesh/issues/706.

Reply:
Okay, that makes sense to me. Thanks for the explanation.